index.html

```diff
@@ -0,0 +1,169 @@
+        </div>
+        <div class="cards">
+            <img src="images/card3.png" alt="">
+            <h1 class="card-title">Multi-Post Stories</h1>
+            <div class="card-text">
+                <h6>CANOPY</h6>
+                <ul>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+            </div>
+            <p class="para">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul>
+                <li class="list">HTML</li>
+                <li class="list">CSS</li>
+                <li class="list">JAVASCRIPT</li>
+            </ul>
+            <button class="btn">See Project</button>
+        </div>
+
+        <div class="cards">
+            <img src="images/card4.png" alt="">
+            <h1 class="card-title">Multi-Post Stories</h1>
+            <div class="card-text">
+                <h6>CANOPY</h6>
+                <ul>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+            </div>
+            <p class="para">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul>
+                <li class="list">HTML</li>
+                <li class="list">CSS</li>
+                <li class="list">JAVASCRIPT</li>
+            </ul>
+            <button class="btn">See Project</button>
+        </div>
+    </div>
+
+    <!-- Myself-->
+    <section class="myself">
+        <h1>About Myself</h1>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of
+            my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+            contact me.</p>
+        <p>LET’S CONNECT</p>
+        <div class="connect">
+            <ul class="socialmedia">
+                <li><a href="#" target="_blank"> <i class="fa-brands fa-twitter icon"></i></a> </li>
+                <li><a href="#" target="_blank"> <i class="fa-brands fa-github icon"></i></a></li>
+                <li><a href="#" target="_blank"><i class="fa-brands fa-linkedin-in icon"></i></a></li>
+                <li><a href="#" target="_blank"></a></li>
+            </ul>
+        </div>
+        <button type="button">Get My Resume</button>
+        <ul class="myself_ul">
+            <li class="language">
+                <h4>Language</h4>
+                <ul>
+                    <li><i class="fa-brands fa-js"></i>JAVASCRIPT</li>
+                    <li><i class="fa-brands fa-html5"></i>HTML</li>
+                    <li><i class="fa-brands fa-css3"></i>CSS</li>
+                </ul>
+            </li>
+            <li>Framework</li>
+           <li> <hr></li>  
+            <li>Skill</li>
+            <li><hr></li>
+        </ul>
+    </section>
```

Comment:
- [ ] Kindly make sure your `About Section` sticks to the Figma design and also add some missing elements like `Chevron-icons`. You can check the **spacing around the section(top && bottom)**, **some missing elements(`Chevron-icons`),** **Correct color on social media icons**,  **Logo of languages(I can suggest you download those images from Figma instead of icons to make it easy)**,... Make sure you check everything and match the design to the Figma template.

| Figma design | Your Design |
| ------------------ | ------------------- |
| ![Screenshot from 2022-07-01 19-17-52](https://user-images.githubusercontent.com/83042742/176942264-5d961374-de89-4710-babc-950049d55457.png) | ![Screenshot from 2022-07-01 19-30-14](https://user-images.githubusercontent.com/83042742/176942685-5bea8698-9612-4494-9d19-b576b83e506a.png) |